packages/techdocs-cli/cypress.config.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { defineConfig } from 'cypress';
+import { initPlugin } from '@frsource/cypress-plugin-visual-regression-diff/plugins';
+
+module.exports = defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      initPlugin(on, config);
+    },
+  },
+  excludeSpecPattern: ['**/__snapshots__/*', '**/__image_snapshots__/*'],
+  env: {
+    mkDocsBaseUrl: 'http://localhost:8000',
+    backstageBaseUrl: 'http://localhost:3000',
+    'cypress-plugin-snapshots': {
+      autoCleanUp: false,
+      imageConfig: {
+        resizeDevicePixelRatio: true,
+        threshold: 0.01,
+      },
+    },
```

Comment:
This looks like config for the old package, I think we can just remove it. There's config for the new one as well (https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff#configuration), but as far as I can tell the defaults should be fine.